ports-wip/bash-static/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# This code Copyright 2012 Todd Mortimer <todd.mortimer@gmail.com>
+#
+# You may do whatever you like with this code, provided the above
+# copyright notice and this paragraph are preserved.
+
+
+set -e
+source ../../lib.sh
+DISTVER="bash-4.4"
+DISTSUFFIX="tar.gz"
+TASK=fetch
+
+DISTFILES="http://ftp.gnu.org/gnu/bash/$DISTVER.$DISTSUFFIX"
+UNPACKCOMD="tar -xf"
+package_init "$@"
+CONFIGURE_CMD="./configure 
+		--enable-static-link
```

Comment:
@berryamin just curious what is the usecase for the statically compiled bash? :smiley:

Reply:
Don't remember exactly but I think my goal at this time was to compile "Term49" with bash as default shell. I managed to compile it but had failure each time I executed it.. for some unknown reason .. Hence this port .. Maybe we can just throw it away .. Term48 is far than enough